b2/account_info.py

```diff
@@ -141,6 +141,42 @@ def clear_large_file_upload_urls(self, file_id):
         pass
 
 
+class UploadUrlPool(object):
```

Comment:
I debated with myself on that.  The tests of the SqliteAccountInfo exercise this code thoroughly.  On the other hand, the code might change later so that's not the case any more.